tools/cldr-apps/js/src/esm/cldrTable.js

```diff
@@ -1759,6 +1270,9 @@ function getValidWinningValue(theRow) {
 }
 
 export {
+  NO_WINNING_VALUE,
```

Comment:
this was surprising to me, winning value shouldn't be null. (Read some of the related code changes but couldn't find a lot of details) In any event, otherwise lgtm.

Reply:
> this was surprising to me, winning value shouldn't be null...

Me too. There's something related, very old, in the 2nd item under Future Discussions in the main agenda. NO_WINNING_VALUE is also in VoteResolver.java, and there's a TODO comment there about it